snapcraft.yaml

```diff
@@ -76,8 +67,8 @@ parts:
       - gstreamer1.0-plugins-good
       - gstreamer1.0-plugins-bad
       - gstreamer1.0-plugins-ugly
-      - libadwaita-1-0
-      - libdbus-1-3
+      #- libadwaita-1-0
```

Comment:
instead of commenting out parts of the yaml these should just be removed if they are not needed anymore?